### ISSUE TITLE
Team Communication

### ISSUE THREAD

**Issue by gpedro** [LFeh/poste-mais#43]
@vitorbritto publicou no Facebook [sobre o Slack](https://www.facebook.com/vitorbritto/posts/10204655801253778) e um rapaz disse que o JusBrasil usa a quase um ano, o que me deixou bastante curioso: que ferramentas vocês usam para como se comunicar com a equipe durante os projetos. Usam integrações e abusam delas? Trás mais comunicação e dinâmica ao projeto?

Eu conheço esses:
* Slack (uso)
* Hipchat (testei)
* CampFire
* gitgitter (uso)

**Comment by jugoncalves** [comment]
Já usei todos. Nunca achei que isso, per se, melhorasse a comunicação nos projetos, mas com certeza facilita. O que eu mais gosto é que carregam `.gifs` automaticamente. :heart:

**Comment by gpedro** [comment]
@jugoncalves olha lá as coisas úteis que o Hubot faz no Slack (:

**Comment by bernardodiasc** [comment]
Uso Slack na empresa que trabalho.

Fato curioso http://valleywag.gawker.com/slack-is-letting-anyone-peek-at-their-competitors-1643790919 :)

**Comment by jugoncalves** [comment]
@gpedro sim, mas se a pessoa não estiver comprometida em ficar olhando, não adianta nada. sem contar que as vezes pode ser muito distrativo essas notificações.

quando eu trabalhava na codeminer, a gente tinha um bot chamado botilda que fazia altas coisas inúteis, mas era divertido.

**Comment by hugobessaa** [comment]
Uso muito o Slack e Email ainda para comunicação online!
